src/io/files.js

```diff
@@ -714,6 +714,88 @@ p5.prototype.httpGet = function () {
  * @param  {function}      [errorCallback] function to be executed if
  *                                    there is an error, response is passed
  *                                    in as first argument
+ *
+ * @example
+ * <div><code>
+ * // Examples use jsonplaceholder.typicode.com for a Mock Data API
+ *
+ * var url = 'https://jsonplaceholder.typicode.com/posts';
+ * var post = { userId: 1, title: 'p5 Clicked!', body: 'p5.js is way cool.' };
```

Comment:
I would prefer another name for the variable just so to not be confused with `httpDo`'s "method" argument